app/controllers/miq_ae_class_controller.rb

```diff
@@ -1029,14 +1029,19 @@ def method_form_fields
 
     if %w[ansible_job_template ansible_workflow_template].include?(location)
       # ManageIQ::Providers::AnsibleTower::Provider.where('zone_id != ?', Zone.maintenance_zone.id)
-      list_of_managers = ManageIQ::Providers::ExternalAutomationManager
-                         .where(:enabled => true)
-                         .pluck(:id, :name)
-                         .map { |r| {:id => r[0], :name => r[1]} }
+      list_of_managers = []
+      ManageIQ::Providers::ExternalAutomationManager.where(:enabled => true).each do |provider|
+        list_of_managers.push({:id => provider.id, :name => provider.name})
+      end
```

Comment:
Not sure why this was changed - the old code doesn't the same thing, so this might be clearer.better:

```suggestion
      list_of_managers = ManageIQ::Providers::ExternalAutomationManager
                         .where(:enabled => true)
                         .pluck(:id, :name)
                         .map { |id, name| {:id => id, :name => name} }
```

Reply:
or without pluck:

```suggestion
      list_of_managers = ManageIQ::Providers::ExternalAutomationManager
                         .where(:enabled => true)
                         .map { |provider| {:id => provider.id, :name => provider.name} }
```